deployer.sh

```diff
@@ -220,6 +221,15 @@ else
       export OPENSHIFT_CA_CRT="$(${SSH_COMMAND} cat /etc/origin/master/ca.crt)"
       export OPENSHIFT_MANAGEMENT_ADMIN_TOKEN="$(${SSH_COMMAND} oc sa get-token -n management-infra management-admin)"
 
+      echo "Assigning alert profiles to Enterprise in ManageIQ"
+      cat <<EOF > profile_assign.sh
+      source /etc/default/evm
+      rails r 'MiqAlertSet.find_by(:guid=>"a16fcf51-e2ae-492d-af37-19de881476ad").assign_to_objects(MiqEnterprise.last); MiqAlertSet.find_by(:guid=>"ff0fb114-be03-4685-bebb-b6ae8f13d7ad").assign_to_objects(MiqEnterprise.last)'
```

Comment:
it would be nice to extract the ruby code to a file of all ruby code executed in the manageiq container